### ISSUE TITLE
Problems Loading the mirror after installing the module

### ISSUE THREAD

**Issue by Oivlis2002** [RaspiManu/MMM-FrameLight#5]
Hello,
I wanted to use the module to controll the frame light for my mirror.
For testing I have connected 4 LEDs of the stripe (WS2801) to the raspberry pi, installed the module and added the example code into the config.js file. I have changed the number of LEDs in the configuration to 4, however after saving and running the file, the mirror didn't start up anymore. Just an empty black screen appeared as if the program was still loading (however still nothing happened after 15minutes of staring at the black screen). Not even the error screen showed up, that you usually get, if there is a problem with the config.js file. If I mark the code section of the FrameLight module in the config.js as a comment everything runs normal again. Hence I suppose there must be a problem within the code of the module. Do you have any idea what could be the issue?
Thanks for your help

**Comment by RaspiManu** [comment]
Dear @Oivlis2002 

sorry for the late response. I am quite busy at the moment. Thank you very much for your interest in our module and telling us about your problem with it. We will do our best to help you out.

Could you please provide us the following information:

- your hardware (Raspberry Pi model or something else)
- your OS with version
- your version of Magic Mirror
- your versions of the module's dependencies
- did you follow all the installation steps correctly? (please double check)
- your config code for the module

Best wishes
RaspiManu

**Comment by Oivlis2002** [comment]
Dear @RaspiManu,
thank you for your response.
Unfortunately I am currently not able to access my Magic Mirror as I am in a different city for studying and I left it at home.
I will probably be back home in May and then provide you with all the information.
In the meantime thank you for your help,
Oivlis2002

**Comment by egarrison89** [comment]
@Oivlis2002 is this similar to what error you are getting?
![image](https://github.com/RaspiManu/MMM-FrameLight/assets/167653521/f0c3227a-002b-440e-af8d-2c18f4da2662)

**Comment by Oivlis2002** [comment]
Hi,
I finally got the chance to work on the mirror again, an now it's working.
I couldn't find the exact issue, because it worked, before I got the chance to replicate the error. I did an update on the MMM-software (I was still using 2.21.0), updated my python versions and deleted and reinstalled the Adafruit CircuitPython WS2801 and the Python Shell repositories. That was enough to get it working, I suppose the error was due to an error during the installation of the Adafruit repositry as I have already used the Python shell succesfully for previous project on the RaspberryPi.
The module is awesome, thanks for your help.

**Comment by Oivlis2002** [comment]
Hi,
I finally got the chance to work on the mirror again, an now it's working.
I couldn't find the exact issue, because it worked, before I got the chance to replicate the error. I did an update on the MMM-software (I was still using 2.21.0), updated my python versions and deleted and reinstalled the Adafruit CircuitPython WS2801 and the Python Shell repositories. That was enough to get it working, I suppose the error was due to an error during the installation of the Adafruit repositry as I have already used the Python shell succesfully for previous project on the RaspberryPi.
The module is awesome, thanks for your help.
